diesel/src/connection/statement_cache.rs

```diff
@@ -108,7 +108,7 @@ pub struct StatementCache<DB: Backend, Statement> {
     pub cache: RefCell<HashMap<StatementCacheKey<DB>, Statement>>,
 }
 
-#[cfg_attr(feature = "clippy", allow(len_without_is_empty, new_without_default_derive))]
+#[cfg_attr(feature = "cargo-clippy", allow(len_without_is_empty, new_without_default_derive))]
```

Comment:
I guess this is set by clippy when you run it ?

Reply:
yes